### PR TITLE
Add profit graphs

### PR DESCRIPTION
**Pull request by utdrmac**
This plugin uses the data collected by the AccountStats plugin to create a visualization of earned lending profits over time.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [x] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by utdrmac** [comment]
@rnevet Can you please test this on polo side?

**Comment by utdrmac** [comment]
screen shot 2017-08-26 at 4 04 34 pm

Just tested on polo side. Sweet. Love it when things work without code modifications. :)

**Comment by utdrmac** [comment]
screen shot 2017-08-26 at 4 07 14 pm
Bitfinex bot charts! Woot!

**Comment by rnevet** [comment]
Cool, i'll Test it.

**Comment by rnevet** [comment]
Could consider indicating which PlugIns are activated in the main json and adding a menu link.

**Comment by utdrmac** [comment]
efa4000

screen shot 2017-08-28 at 4 44 53 pm

**Comment by rnevet** [comment]
You could add to the lendingbot.html menu:
'''
                    <li data-toggle="collapse" data-target=".navbar-collapse.in">
                        <a href="chart.html">
                            <i class="glyphicon glyphicon-stats"></i>
                            Chart
                        </a>
                    </li>
'''

And also the link back in the chart.html

File: www/chart.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+          var options = {
+              'title': 'Lending Daily Profits',
+              'width': 1024,
+              'height': 512,
+              'vAxes': { 
+                  0: { 'title': 'Daily' },
+                  1: { 'title': 'Totals' }
+              },
+              'series': axisTargets
```
Comment (rnevet) [inline www/chart.html]: Would add:
```
'explorer':{}
```

To allow zoom

**Comment by utdrmac** [comment]
Re: Adding to the menu: Yes, I don't want to add it statically. The idea of adding the plugin sections to botlog is so the lendingbot.html page will generate links to <pluginname>.html if so added. Will have to create a standard for that.

Re: Totals: Yes, will fix totals by converting to BTC

Re: Graph changes: Will add recommendation to get zooming.

**Comment by utdrmac** [comment]
Depends on #497

**Comment by utdrmac** [comment]
@rnevet updated to be per-coin graphs. let me know what you think.

**Comment by rnevet** [comment]
Looks nice.
Coin order is bothering me, additionally on mobile the graphs are quite big.

**Comment by utdrmac** [comment]
What do you mean by "coin order"? It's whatever order you get from SELECT DISTINCT currency... guess I can put an ORDER BY to make it alphabetical?

Ah. Didn't even think about mobile. Never used this on mobile; only laptop. Any ideas on how to scale for mobile?

**Comment by rnevet** [comment]
@utdrmac 
regarding mobile, it looks not to bad, however width is fixed to 1024 - maybe could change that.
https://developers.google.com/chart/interactive/docs/basic_customizing_chart

**Comment by rnevet** [comment]
Regarding the mobile look:
removing height and width from the graph options and setting the following style on the div works nicely I think:
'''
<div id="' + chartDiv + '" style="width: 100%; height: 420px">
'''

**Comment by rnevet** [comment]
Ordering didn't work.

And last thing would be adding the menu item, I think then we can merge.

**Comment by rnevet** [comment]
I know why the sorting doesn't work, it's because on javascript side the jsonHistory is an object and not a list, going over the object keys is done in random order.

**Comment by utdrmac** [comment]
Should be good now

**Pull request merged**